pkg/controller/perconaservermongodbbackup/perconaservermongodbbackup_controller.go

```diff
@@ -209,26 +217,35 @@ func (r *ReconcilePerconaServerMongoDBBackup) reconcile(
 		return status, errors.Wrap(err, "failed to run backup")
 	}
 
-	cjobs, err := backup.HasActiveJobs(ctx, r.newPBMFunc, r.client, cluster, backup.NewBackupJob(cr.Name), backup.NotPITRLock)
-	if err != nil {
-		return status, errors.Wrap(err, "check for concurrent jobs")
-	}
-
-	if cjobs {
-		if cr.Status.State != psmdbv1.BackupStateWaiting {
-			log.Info("Waiting to finish another backup/restore.")
-		}
-		status.State = psmdbv1.BackupStateWaiting
-		return status, nil
-	}
-
-	if cr.Status.State == psmdbv1.BackupStateNew || cr.Status.State == psmdbv1.BackupStateWaiting {
+	switch cr.Status.State {
+	case psmdbv1.BackupStateNew, psmdbv1.BackupStateWaiting:
 		time.Sleep(10 * time.Second)
 		return bcp.Start(ctx, r.client, cluster, cr)
+	case psmdbv1.BackupStateRunning:
+	default:
+		cjobs, err := backup.HasActiveJobs(ctx, r.newPBMFunc, r.client, cluster, backup.NewBackupJob(cr.Name), backup.NotPITRLock)
+		if err != nil {
+			return status, errors.Wrap(err, "check for concurrent jobs")
+		}
+
+		if cjobs {
+			if cr.Status.State != psmdbv1.BackupStateWaiting {
+				log.Info("Waiting to finish another backup/restore.")
+			}
+			status.State = psmdbv1.BackupStateWaiting
+			return status, nil
+		}
 	}
 
 	time.Sleep(5 * time.Second)
-	return bcp.Status(ctx, cr)
+
+	err := retry.OnError(defaultBackoff, func(err error) bool { return err != nil }, func() error {
```

Comment:
Since the 1st retry will be executed after 5 seconds due to the `time.Sleep(5 * time.Second)` we have already, maybe we can adjust the default backoff config to start after 10 seconds. With the current config we will start 1 retry after 5 seconds, then again after 5 seconds and then after 10. WDYT?